### PR TITLE
Merge Header branch - Navigation bar, hero

### PR DESCRIPTION
**Pull request by Gina-Frankel**
- Add blank navigation bar
- Add Hero with heading, subheading and image
- Added styling
- Call to action ("Find Out More" button) not added as not yer created

### CODE REVIEW

**Comment by Gina-Frankel** [comment]
Note: Call to action ("Find out More") component not added as needs to be created first.

File: public/index.html [hunk 1]
```diff
@@ -3,18 +3,17 @@
   <head>
     <meta charset="utf-8" />
     <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <!-- <link href="https://fonts.googleapis.com/css2?family=Permanent+Marker&display=swap" rel="stylesheet"> -->
```
Comment (hturnbull93) [inline public/index.html:6]: is this being used?

File: src/components/App.js [hunk 2]
```diff
@@ -1,11 +1,27 @@
...
+      <div className="Hero-container">
+        <div className="row justify-content-around">
+          <div className="col-lg-6 text-center">
+          <Hero/>
+          </div>
+          <div className="col-lg-5 text-center">
+          <HeroPicture/>
+          </div>
+        </div>
+      </div>
```
Comment (hturnbull93) [inline src/components/App.js:24]: Why is the HeroPicture separate from Hero? this ` <div className="Hero-container">` should probably be the component, no?

File: src/components/HeroPicture.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+function HeroPicture() {
+  return (
+    <div className="hero-picture">
+    
+    <img src="./statue.png" alt="Generic problematic statue"></img>
+   
+    </div>
+  );
+}
```
Comment (hturnbull93) [inline src/components/HeroPicture.js:12]: Why a component just to render an img? This probably should be in one component with the text in the Hero

File: src/components/App.js [hunk 4]
```diff
@@ -1,11 +1,27 @@
...
   return (
-    <div className="App">
-      <header className="App-header">
+    <div className="app">
+      <header className="app-header">
+        <NavBar/>
       </header>
+      <div className="Hero-container">
+        <div className="row justify-content-around">
+          <div className="col-lg-6 text-center">
```
Comment (hturnbull93) [inline src/components/App.js:17]: Perhaps take a look at the Bootstrap Jumbotron for styling the Hero?

File: src/components/NavBar.css [hunk 5]
```diff
@@ -0,0 +1,8 @@
+.NavBar{
+  background-color: #BB0032;
+  color: white;
+}
+
+a {
+  color: white !important;
+}
```
Comment (hturnbull93) [inline src/components/NavBar.css:8]: this will target `a`s sitewide, is that intended?

File: src/components/NavBar.js [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+    <div className="NavBar">
+    <nav className="navbar navbar-expand-lg heading">
+      <button className="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNavAltMarkup" aria-controls="navbarNavAltMarkup" aria-expanded="false" aria-label="Toggle navigation">
+        <span className="navbar-toggler-icon"></span>
+      </button>
+      <div className="collapse navbar-collapse" id="navbarNavAltMarkup">
+        
+      </div>
+    </nav>
+    </div>
```
Comment (hturnbull93) [inline src/components/NavBar.js:16]: This seems unfinished, no links?

File: src/components/App.js [hunk 7]
```diff
@@ -1,11 +1,27 @@
...
+          <Hero/>
+          </div>
+          <div className="col-lg-5 text-center">
+          <HeroPicture/>
+          </div>
```
Comment (hturnbull93) [inline src/components/App.js:22]: no indentation here? has this been linted?

File: src/components/App.js [hunk 8]
```diff
@@ -1,11 +1,27 @@
 import React from 'react';
+import NavBar from './NavBar.js'
+import Hero from './Hero.js'
+import HeroPicture from './HeroPicture.js'
 import StatueListContainer from './StatueListContainer.js'
+import './App.css';
+import '../index.css';
```
Comment (hturnbull93) [inline src/components/App.js:7]: Some semicolons, some without, has this been linted?

File: src/components/Hero.js [hunk 9]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+
+function Hero() {
+  return (
+    <div className="Hero">
```
Comment (hturnbull93) [inline src/components/Hero.js:6]: is `.Hero` targeted in css? Also, generally HTML class names are not capitalised.

File: src/components/Hero.js [hunk 10]
```diff
@@ -0,0 +1,15 @@
...
+    
+    <h1>Don't stand for art that doesn't stand for you</h1>
+    <p>There are dozens of public artworks depicting people involved in slavery</p>
+   
+    </div>
```
Comment (hturnbull93) [inline src/components/Hero.js:11]: Perhaps use indentation rather than empty line spaces here (and elsewhere)?

File: src/components/NavBar.js [hunk 11]
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import './NavBar.css';
+
+
+function NavBar() {
+  return (
+    <div className="NavBar">
+    <nav className="navbar navbar-expand-lg heading">
+      <button className="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNavAltMarkup" aria-controls="navbarNavAltMarkup" aria-expanded="false" aria-label="Toggle navigation">
```
Comment (hturnbull93) [inline src/components/NavBar.js:9]: this line is really long

File: src/components/Hero.spec.js [hunk 12]
```diff
@@ -0,0 +1,18 @@
...
+    const wrapper = shallow(<Hero/>);
+    
+    expect(wrapper.text()).toContain("Don't stand for art that doesn't stand for you")
+  
+  });
+  it ('displays tag line', () => {
+    const wrapper = shallow(<Hero/>);
+    
+    expect(wrapper.text()).toContain("There are dozens of public artworks depicting people involved in slavery")
+
```
Comment (hturnbull93) [inline src/components/Hero.spec.js:16]: fair tests, though perhaps brittle if we want to change the copy?

File: src/components/App.css [hunk 13]
```diff
@@ -0,0 +1,22 @@
...
+
+
+img {
+  max-width: 100%;
+  height: auto;
+}
+
+.landing-page {
+  margin-block-start: 2em;
+  margin-block-end: 2em;
```
Comment (hturnbull93) [inline src/components/App.css:21]: Perhaps look at `margin` css property? `margin-block` is particularly for when you'er using some odd non right to left writing mode and has less browser support.

File: src/components/App.css [hunk 14]
```diff
@@ -0,0 +1,22 @@
...
+  font-family: 'Edo-sz';
+}
+
+
+img {
+  max-width: 100%;
+  height: auto;
+}
+
+.landing-page {
```
Comment (hturnbull93) [inline src/components/App.css:19]: I can't find where this class is applied.

File: src/components/HeroPicture.js [hunk 15]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+
+function HeroPicture() {
+  return (
+    <div className="hero-picture">
+    
+    <img src="./statue.png" alt="Generic problematic statue"></img>
```
Comment (hturnbull93) [inline src/components/HeroPicture.js:8]: Better to import the image in JS, or use an absolute path, if this component was used on another page the src would look relative to that page and would 404

File: src/components/App.css [hunk 16]
```diff
@@ -0,0 +1,22 @@
+
+.app {
+  background-color: #0E0E0E;
+  color: white;
+  font-family: 'Montserrat', sans-serif;
+  letter-spacing: 2px;
```
Comment (hturnbull93) [inline src/components/App.css:6]: Why letter spacing?

File: src/components/index.js [hunk 17]
```diff
@@ -0,0 +1 @@
+export {StatueListContainer} from './StatueListContainer'
```
Comment (hturnbull93) [inline src/components/index.js:1]: I've never seen this before, what is it for?

File: src/components/App.js [hunk 18]
```diff
@@ -1,11 +1,27 @@
 import React from 'react';
+import {NavBar} from './NavBar.js'
+import {Hero} from './Hero.js'
+import {HeroPicture} from './HeroPicture.js'
 import StatueListContainer from './StatueListContainer.js'
```
Comment (benfurber) [inline src/components/App.js:5]: You've created the index so you don't have to do this deep linking.

File: src/components/index.js (hunk 17 above)
Comment (benfurber) [inline src/components/index.js:1]: The other components you've created should be here too.

File: src/index.css [hunk 19]
```diff
@@ -0,0 +1,4 @@
+@font-face {
+  font-family: 'Edo-sz';
+  src: local('Edo-sz'), url(./fonts/edosz.ttf) format('truetype');
+}
```
Comment (benfurber) [inline src/index.css:4]: Line break.

File: src/index.js [hunk 20]
```diff
@@ -2,6 +2,7 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import App from './components/App';
 import 'bootstrap/dist/css/bootstrap.css';
+import './fonts/edosz.ttf'
```
Comment (benfurber) [inline src/index.js:5]: Shouldn't this be a link to the css file not the font file itself?

File: src/index.js [hunk 21]
```diff
@@ -2,6 +2,7 @@ import React from 'react';
...
 // import * as serviceWorker from './serviceWorker';
```
Comment (benfurber) [inline src/index.js:6]: If this has been removed let's remove the comment.

File: src/components/Hero.js [hunk 22]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
```
[the remark this answers is not in the document]
Comment (Gina-Frankel) [reply src/components/Hero.js:1]: I had not linted it, but now Prettier is set to lint on save :)

File: src/components/App.js (hunk 18 above)
Comment (Gina-Frankel) [reply src/components/App.js:5]: Is this what it should be like?
`import { Hero, NavBar } from "./index.js";`

File: src/components/Hero.js [hunk 23]
```diff
@@ -0,0 +1,15 @@
...
+
+
+export function Hero() {
+  return (
+    <div className="Hero">
+    
+    <h1>Don't stand for art that doesn't stand for you</h1>
```
[the remark this answers is not in the document]
Comment (Gina-Frankel) [reply src/components/Hero.js:8]: That is a good idea.
- Would I store them as a JS object.

I was thinking something like this: 

const labels = [ 
  {
    heroHeading: "Don't stand for art that doesn't stand for you"
  },
  {
    heroSubHeading: "Joseph Edgar Boehm"
  }
]

File: src/components/index.js (hunk 17 above)
Comment (Gina-Frankel) [reply src/components/index.js:1]: It is a way that you can centralise all your exports.
 Having named exports enables you to do this and it makes it easier to manage all your exports.

File: src/components/App.css (hunk 14 above)
Comment (Gina-Frankel) [reply src/components/App.css:19]: That was the previous name of the class, it has now been changed to the correct name.

File: src/components/App.css (hunk 16 above)
Comment (Gina-Frankel) [reply src/components/App.css:6]: We thought it looked a bit squashed - happy to delete it and we can decide together at a later point

File: src/components/NavBar.css (hunk 5 above)
Comment (Gina-Frankel) [reply src/components/NavBar.css:8]: Removed the whole thing for now, as the links are not in, and will keep what you wrote in mind when I put the links in.

File: src/components/Hero.js (hunk 23 above)
[the remark this answers is not in the document]
Comment (benfurber) [reply src/components/Hero.js]: Yes, something like that.

File: src/components/App.js (hunk 18 above)
Comment (benfurber) [reply src/components/App.js]: Yes.

**Pull request merged**